src/routes/intro/IntroContainer.js

```diff
@@ -0,0 +1,195 @@
+// @flow
+
+import { translate, type TFunction } from 'react-i18next'
+import * as React from 'react'
+import type { NavigationScreenProp } from 'react-navigation'
+import Language from './assets/Language.svg'
+import Offers from './assets/Offers.svg'
+import Search from './assets/Search.svg'
+import Events from './assets/Events.svg'
+import type { ThemeType } from '../../modules/theme/constants/theme'
+import withTheme from '../../modules/theme/hocs/withTheme'
+import { Switch, FlatList, Dimensions, Platform, PermissionsAndroid } from 'react-native'
+import styled, { type StyledComponent } from 'styled-components/native'
+import AppSettings from '../../modules/settings/AppSettings'
+import SettingItem from '../settings/components/SettingItem'
+import SlideContent, { type SlideContentType } from './SlideContent'
+import SentryIntegration from '../../modules/app/SentryIntegration'
+import SlideFooter from './SlideFooter'
+import type { ViewToken } from 'react-native/Libraries/Lists/ViewabilityHelper'
+import Geolocation from '@react-native-community/geolocation'
+
+const Container: StyledComponent<{ width: number }, {}, *> = styled.View`
+  display: flex;
+  flex-direction: column;
+  width: ${props => props.width};
+  height: 100%;
+  justify-content: space-between;
+`
+
+const ImageContent = styled.Image`
+  justify-content: center;
+  align-self: center;
+  display: flex;
+  height: 100%;
+  width: 60%;
+  resize-mode: contain
+`
+
+type PropsType = {| t: TFunction, navigation: NavigationScreenProp<*>, theme: ThemeType |}
+type StateType = {|
+  slideCount: number,
+  currentSlide: number,
+  allowPushNotifications: boolean,
+  useLocationAccess: boolean,
+  allowSentry: boolean,
+  width: number
+|}
+
+class Intro extends React.Component<PropsType, StateType> {
+  _appSettings: AppSettings
+  _flatList: {current: null | React$ElementRef<typeof FlatList>}
+
+  constructor (props: PropsType) {
+    super(props)
+    this.state = {
+      slideCount: this.slides().length,
+      currentSlide: 0,
+      allowPushNotifications: false,
+      useLocationAccess: false,
+      allowSentry: false,
+      width: Dimensions.get('window').width
```

Comment:
Does rotating the device work while the intro screens are shown?

Reply:
Good point, it works, but looks bad. Should it be possible to rotate the screen?

Reply:
I'd say no :D

Reply:
Nevermind, I just fixed it :)